### ISSUE TITLE
Preset icon matching

### ISSUE THREAD

**Issue by LanesGood** [developmentseed/observe#279]
Despite the fixes in #230 it still appears that preset icons are incorrect.

This is on Android and iPhone, running locally on emulator, and using the latest release apk on an Android
Ex:
<img width="373" alt="Screen Shot 2020-05-29 at 5 32 53 PM" src="https://user-images.githubusercontent.com/12634024/83307318-736dc800-a1d2-11ea-8bc3-a80d1ca2644f.png">
<img width="373" alt="Screen Shot 2020-05-29 at 5 31 20 PM" src="https://user-images.githubusercontent.com/12634024/83307228-3f92a280-a1d2-11ea-9f95-27233b0001c5.png">
<img width="373" alt="Screen Shot 2020-05-29 at 5 32 15 PM" src="https://user-images.githubusercontent.com/12634024/83307277-5e913480-a1d2-11ea-9367-05b88a7e37de.png">

**Comment by geohacker** [comment]
Oh so weird. I'm not able to reproduce this. @vgeorge could you take a look?

**Comment by vgeorge** [comment]
@LanesGood tested on both platforms, presets look good to me on branch `edit-ways`.

**Comment by LanesGood** [comment]
@vgeorge @geohacker I'm wiping emulator data often - any other cache cleaning I need to be doing?

**Comment by sethvincent** [comment]
I've also been seeing the types of icon issues shown in these screenshots. I'm not sure why it would be different for some of us.

**Comment by LanesGood** [comment]
@geohacker @sethvincent I've deleted and  re-cloned the repo, spun up a new device in emulator, and I'm still seeing this on android with the `edit-ways` branch

<img src="https://user-images.githubusercontent.com/12634024/83551208-0ca02580-a4d6-11ea-91f4-f69be841d00e.png" width="240"/>
<img src="https://user-images.githubusercontent.com/12634024/83551465-6143a080-a4d6-11ea-809c-4f3692170802.png" width="240"/>

Noting that map icons do appear correct:
<img src="https://user-images.githubusercontent.com/12634024/83551577-7f110580-a4d6-11ea-9f8b-662672ad8ca6.png" width="240"/>



<details>
<summary>The results of `console.log(filteredPresets)` don't offer much insight here:</summary>

```
 [ { icon: 'maki-home',
                             │ fields: [ 'name', 'building', 'levels', 'height', 'address' ],
                             │ moreFields:
                             │ [ 'architect',
                             │ 'building/material',
                             │ 'layer',
                             │ 'roof/colour',
                             │ 'smoking',
                             │ 'wheelchair' ],
                             │ geometry: [ 'point', 'area' ],
                             │ tags: { building: '*' },
                             │ matchScore: 0.6,
                             │ terms: [],
                             │ name: 'Building',
                             │ key: 'building' },
                             │ { icon: 'maki-bus',
                             │ fields: [ 'name', 'network', 'operator', 'bench', 'shelter' ],
                             │ geometry: [ 'point', 'vertex' ],
                             │ tags: { highway: 'bus_stop' },
                             │ name: 'Bus Stop / Platform',
                             │ searchable: false,
                             │ key: 'highway/bus_stop' },
                             │ { fields: [ 'name' ],
                             │ geometry: [ 'point' ],
                             │ tags: {},
                             │ name: 'Point',
                             │ matchScore: 0.1,
                             │ key: 'point' },
                             │ { icon: 'maki-restaurant',
                             │ fields:
                             │ [ 'name',
                             │ 'cuisine',
                             │ 'address',
                             │ 'building_area',
                             │ 'opening_hours',
                             │ 'outdoor_seating' ],
                             │ moreFields:
                             │ [ 'takeaway',
                             │ 'delivery',
                             │ 'capacity',
                             │ 'diet_multi',
                             │ 'smoking',
                             │ 'internet_access',
                             │ 'internet_access/fee',
                             │ 'internet_access/ssid',
                             │ 'website',
                             │ 'phone',
                             │ 'email',
                             │ 'fax',
                             │ 'wheelchair',
                             │ 'bar' ],
                             │ geometry: [ 'point', 'area' ],
                             │ terms:
                             │ [ 'bar',
                             │ 'breakfast',
                             │ 'cafe',
                             │ 'café',
                             │ 'canteen',
                             │ 'coffee',
                             │ 'dine',
                             │ 'dining',
                             │ 'dinner',
                             │ 'drive-in',
                             │ 'eat',
                             │ 'grill',
                             │ 'lunch',
                             │ 'table' ],
                             │ tags: { amenity: 'restaurant' },
                             │ name: 'Restaurant',
                             │ key: 'amenity/restaurant' },
                             │ { icon: 'maki-school',
                             │ fields:
                             │ [ 'name',
                             │ 'operator',
                             │ 'address',
                             │ 'religion',
                             │ 'denomination',
                             │ 'website' ],
                             │ moreFields:
                             │ [ 'internet_access',
                             │ 'internet_access/ssid',
                             │ 'phone',
                             │ 'email',
                             │ 'fax',
                             │ 'wheelchair' ],
                             │ geometry: [ 'point', 'area' ],
                             │ terms: [ 'academy', 'elementary school', 'middle school', 'high school' ],
                             │ tags: { amenity: 'school' },
                             │ name: 'School',
                             │ key: 'amenity/school' },
                             │ { icon: 'maki-park',
                             │ fields: [ 'leaf_type_singular', 'leaf_cycle_singular', 'denotation' ],
                             │ geometry: [ 'point', 'vertex' ],
                             │ tags: { natural: 'tree' },
                             │ terms: [],
                             │ name: 'Tree',
                             └ key: 'natural/tree' } ]
```

</details>

No issues on iPhone simulator as seen below:

<img src="https://user-images.githubusercontent.com/12634024/83554369-a4077780-a4da-11ea-9a1a-709d4632f3de.png" width="240"/>

**Comment by LanesGood** [comment]
As an update:

I checked in the `develop` branch and get the same icons on Select Feature Type, and elsewhere also have incorrect icons.

Following @geohacker link to [this issue](https://github.com/oblador/react-native-vector-icons/issues/276), I looked around and believe the edits to the `build.gradle` file may hold the solution: 
https://github.com/oblador/react-native-vector-icons#android

However, I'm definitely not implementing this correctly as it has no effect.

**Comment by geohacker** [comment]
Fixed in #300. This came down to manually updating the ttf file in the android assets directory. This is not a problem for iOS.
